src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -843,7 +843,7 @@ protected void removeWarFileLines(List<String> dockerfileLines) throws PluginExe
                 // The command must be to the left of any comments.
                 String[] cmdSegments = trimLine.split("#")[0].split("\\s+");
                 // if the line starts with COPY and the second to last segment ends with ".war", it is a WAR file COPY line
-                if (cmdSegments[0].equalsIgnoreCase("COPY")) {
+                if (cmdSegments[0].equalsIgnoreCase("COPY") || cmdSegments[0].equalsIgnoreCase("ADD")) {
                     if (cmdSegments.length < 3) {
                         throw new PluginExecutionException("Incorrect syntax on this line in the Dockerfile: '" + line + 
                         "'. There must be at least two arguments for the COPY command, a source path and a destination path.");
```

Comment:
The exception message should be updated as well.

Reply:
ok